docs/root/intro/version_history.rst

```diff
@@ -3,6 +3,7 @@ Version history
 
 1.14.0 (Pending)
 ================
+* config: use type URL to select an extension whenever the config type URL (or its previous versions) uniquely identify a typed extension.
```

Comment:
can you ref link to the new doc page you added?

Reply:
Done.